build.gradle

```diff
@@ -284,7 +284,7 @@ test {
     exclude "**/SplitInputStreamTest.class"
     exclude "**/JPF_*.class"
 
-    jvmArgs += ['--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED', '--add-opens', 'java.base/jdk.internal.misc=ALL-UNNAMED', '--add-opens', 'java.base/java.lang=ALL-UNNAMED', '--add-opens', 'java.base/java.util=ALL-UNNAMED']
+    jvmArgs += ['--add-exports', 'java.base/jdk.internal.access=ALL-UNNAMED', '--add-opens', 'java.base/jdk.internal.access=ALL-UNNAMED', '--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED', '--add-opens', 'java.base/jdk.internal.misc=ALL-UNNAMED', '--add-opens', 'java.base/java.lang=ALL-UNNAMED', '--add-opens', 'java.base/java.util=ALL-UNNAMED']
```

Comment:
'java.base/jdk.internal.access=ALL-UNNAMED' seems to be repeated here.

Reply:
`'--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED',`
` '--add-opens', 'java.base/jdk.internal.misc=ALL-UNNAMED',`

Two statements were written for making them(`java.base/jdk.internal.misc`) accessible to other modules. Same approach I applied for `java.base/jdk.internal.access` like below

```
'--add-exports', 'java.base/jdk.internal.access=ALL-UNNAMED',
 '--add-opens', 'java.base/jdk.internal.access=ALL-UNNAMED', 

'--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED', 
'--add-opens', 'java.base/jdk.internal.misc=ALL-UNNAMED',
```